jetty-ee10/jetty-ee10-quickstart/src/main/java/org/eclipse/jetty/ee10/quickstart/QuickStartDescriptorProcessor.java

```diff
@@ -185,13 +185,11 @@ public void visitContextParam(WebAppContext context, Descriptor descriptor, XmlP
                 Object o = context.getAttribute(MetaInfConfiguration.METAINF_TLDS);
                 if (o instanceof Collection<?>)
                     tlds.addAll((Collection<?>)o);
+
                 for (String i : values)
                 {
-                    Resource r = Resource.newResource(normalizer.expand(i));
-                    if (r.exists())
-                        tlds.add(r.getURI().toURL());
-                    else
-                        throw new IllegalArgumentException("TLD not found: " + r);
+                    String entry = normalizer.expand(i);
+                    tlds.add(Resource.toURI(entry).toURL());
                 }
 
```

Comment:
The METAINF_RESOURCES below are going to need to be mounted and those mounts released.  So more work for #8280